source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -211,54 +164,61 @@ If you choose the `flannel` add-on, then you must add the following to the
 
     --pod-network-cidr 10.244.0.0/16
 
-If you are using CRI-O and `flannel` and you want to use Kata Containers, edit
-the :file:`/etc/crio/crio.conf` file to add:
+If you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
```

Comment:
i suspect this will apply outside of flannel too, but i am guessing flannel is the one you have tested and hence the statement?

Reply:
Yes, we're restricting this tutorial's plugin to `flannel` only. @eadamsintel and I plan to do a mapping session for K8S tutorials during WW 51--during which we'll discuss which CNIs to incorporate and where in docs. Perhaps you'd like to call into that meeting?

Reply:
would love to be there. Thanks @mvincerx